connect.go

```diff
@@ -0,0 +1,97 @@
+package breez
+
+import (
+	"context"
+	"io"
+	"sync/atomic"
+
+	"github.com/breez/breez/data"
+	"github.com/breez/lightninglib/lnrpc"
+)
+
+var (
+	connectedToRoutingNode int32
+)
+
+/*
+ConnectAccount force connect to the routing node.
+This is meant to be used from the mobile platform when the network and device state allows it.
+*/
+func ConnectAccount() error {
+	return connectRoutingNode()
+}
+
+/*
+IsConnectedToRoutingNode returns the connection status to the routing node
+*/
+func IsConnectedToRoutingNode() bool {
+	return atomic.LoadInt32(&connectedToRoutingNode) == 1
+}
+
+func watchRoutingNodeConnection() error {
+	log.Infof("watchRoutingNodeConnection started")
+	subscription, err := lightningClient.SubscribePeers(context.Background(), &lnrpc.PeerSubscription{})
+	if err == io.EOF {
```

Comment:
Why a special return for io.EOF when the next if statement is going to return all the same?

Reply:
Right!